lib/agent0/agent0/utilities/docker.py

```diff
@@ -0,0 +1,60 @@
+"""Utilities associated to manipulating Docker."""
+import subprocess
+import logging
+import os
+import time
+from pathlib import Path
+
+
+def check_docker(restart: bool = False) -> None:
+    """Check whether docker is running to your liking.
+
+    Parameters
```

Comment:
Arguments

Reply:
numpydoc says Parameters https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
Yes, this is the one exception we have made from the numpy doc